docs/swagger.yaml

```diff
@@ -609,6 +609,75 @@ paths:
           description: Unsupported media type
         '500':
           description: Something went wrong from Vipps Server side
+  '/ecomm/v2/payments/{orderId}/approve':
+    post:
+      tags:
+        - Vipps eCom API
+      summary: 'Force Approve Payment '
+      description: >-
+        The force approve endpoint allows developers to approve a payment through the Vipps Ecom API
+        without the use of the Vipps App. This is useful for automated testing. The endpoint is 
+        only available in our Test environment. Attempted use of the endpoint in production will 
+        not be tollerated.
```

Comment:
Only one 'l' in 'tolerated'.

Reply:
We have used "Vipps eCom API" (lowercase e, capital C) in other places.

Reply:
Both fixed